parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -199,6 +199,13 @@ if [ "${NEEDS_DOCKER_LOGIN}" == "true" ]; then
     set -x
 fi
 
+# Before setting up kubelet, ensure IMDS restriction iptables rules so that all Pods can get desired IMDS access
```

Comment:
just for my understanding, is the only requirement here that we perform this step before we start the kubelet service?

Reply:
yep, only need to ensure the rule is added before starting kubelet.